driver/TrEE/TrEE/OpteeVariableService.h

```diff
@@ -23,12 +23,10 @@ UEFI Variable Services definitions for accessing operations.
 typedef enum _VARIABLE_SERVICE_OPS
 {
     VSGetOp = 0,
+    VSGetNextVarOp,
```

Comment:
Why have these rearranged?

Reply:
They are in the same order as the UEFI spec now (plus exit boot services).